### DISCUSSION TITLE
Choose a license

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#13] [General]
A public Github should have a open source license implemented. Please discuss and choose which one we want to implement:

https://choosealicense.com/

**Comment by basvansambeek** [comment]
I found these descriptions more informative then choosealicense.com: https://gist.github.com/nicolasdao/a7adda51f2f185e8d2700e1573d8a633 and https://exygy.com/blog/which-license-should-i-use-mit-vs-apache-vs-gpl/

To me it seems what we want to decide first is if we want to maximize **freedom** (“I don’t care what you do, just don’t sue me”/“Hey, it’s free, no legal restrictions, why not try it out?”), or maximize **reciprocation** (“I open sourced my code so you should too”).

For this GitHub alone it might not matter that much, but perhaps for whatever the treasure-to-be might fund, it is important to think about what license it would require/prefer.

The “freedom” licenses facilitate easier adoption; there’s no restriction on using it, nor do you have to share whatever you made with others.  It will make commercialization easier.

The “reciprocation” licenses require you to open source your code, to the greater benefit of the community, which will have more tools available to it. It does however make it trickier to use, because there are conditions connected to using it that you have to accept.

I’m not a lawyer, and there are some legal aspects to that I can’t judge, but I do think it’s important to at least think about what we want the spirit of this community to be. The decisions we make here will greatly influence the future of our community. (Not saying that we should dictate a license, but the utility of the various licenses is quite different.)

**Reply by rophiroth** [reply]
> I found these descriptions more informative then choosealicense.com: https://gist.github.com/nicolasdao/a7adda51f2f185e8d2700e1573d8a633 and https://exygy.com/blog/which-license-should-i-use-mit-vs-apache-vs-gpl/
> 
> To me it seems what we want to decide first is if we want to maximize **freedom** (“I don’t care what you do, just don’t sue me”/“Hey, it’s free, no legal restrictions, why not try it out?”), or maximize **reciprocation** (“I open sourced my code so you should too”).
> 
> For this GitHub alone it might not matter that much, but perhaps for whatever the treasure-to-be might fund, it is important to think about what license it would require/prefer.
> 
> The “freedom” licenses facilitate easier adoption; there’s no restriction on using it, nor do you have to share whatever you made with others. It will make commercialization easier.
> 
> The “reciprocation” licenses require you to open source your code, to the greater benefit of the community, which will have more tools available to it. It does however make it trickier to use, because there are conditions connected to using it that you have to accept.
> 
> I’m not a lawyer, and there are some legal aspects to that I can’t judge, but I do think it’s important to at least think about what we want the spirit of this community to be. The decisions we make here will greatly influence the future of our community. (Not saying that we should dictate a license, but the utility of the various licenses is quite different.)

Thank you, according to what you say, reciprocation sounds great, what licences are of that kind?
